bigchaindb/__init__.py

```diff
@@ -45,7 +45,22 @@
         'private': None,
     },
     'keyring': [],
-    'backlog_reassign_delay': 120
+    'backlog_reassign_delay': 120,
+    'log': {
```

Comment:
If we are not using this, it should be removed.

If we want to use it we should put in here the defaults and load them at start time with the rest of the configuration.
I don't see a need to do this at this point. Maybe create an issue for this.